recipe_Menager.py

```diff
@@ -0,0 +1,42 @@
+class RecipeManager:
+    def __init__(self):
+        self.recipes = []
+
+    def add_recipe(self, recipe):
+        self.recipes.append(recipe)
+
+    def update_recipe(self, recipe_index, new_recipe):
+        self.recipes[recipe_index] = new_recipe
+      #on this point I am confused, because I can see 2 meanings:
+      # 1. change older recipe for new one and I had choosen this version
+      # 2. change the integrent in recipe
+
+    def delete_recipe(self, recipe_index):
+        self.recipes.pop(recipe_index)
+
+    def display_recipes(self):
+        for recipe in self.recipes:
+            print(recipe)
```

Comment:
Its probable that the recipe will need its own format form which will associate with graphical interface at the end so I think just printing at this point might not work in the end (a comment needs to be put in at least). Its quite possible that we will end up with a separate file/class for the main loop and the graphics which will do all the work of formatting.